src/components/SkillCardScrollList/SkillCardScrollList.js

```diff
@@ -18,12 +18,36 @@ class SkillCardScrollList extends Component {
     this.state = {
       cards: [],
       skills: this.props.skills,
+      scrollCards: 3,
     };
   }
 
-  componentDidMount() {
+  componentDidMount = () => {
     this.loadSkillCards();
-  }
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions);
+
+    $('.scrolling-wrapper').css({ width: '104%' });
+  };
+
+  componentWillUnmount = () => {
+    window.removeEventListener('resize', this.updateWindowDimensions);
+  };
+
+  updateWindowDimensions = () => {
+    this.setState({
+      /* eslint-disable no-nested-ternary */
+      scrollCards:
+        window.innerWidth > 1400
```

Comment:
Remove `disable eslint` and go through this (https://stackoverflow.com/questions/12984191/alternative-for-multiple-if-statements) to avoid warnings/error.